### ISSUE TITLE
Is GIPHY Clips (GIFs with Sound) Supported ?

### ISSUE THREAD

**Issue by maheshperi19** [amantoux/giphy_selector#2]
Giphy documentation says "Clips, Emoji, Text, and GIPHY Text content types are only available through the SDK." Are you supporting "GIPHY Clips (GIFs with Sound)" ?

**Comment by amantoux** [comment]
No, but to be honest, I discovered their exists with your issue, I can add support if need be

For reference:
https://developers.giphy.com/docs/clips/endpoint#trending

**Comment by maheshperi19** [comment]
Yes Please. I thought to use Flutter package instead of calling thier Giphy API's separately. So found this flutter package. Please add clips (gifs with sound). So that it would use for many developers.

Thanks.
Mahesh.
